samples/errors6.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+// DOES NOT COMPILE
+// used only to illustrates a pattern
```

Comment:
Code that doesn't compile isn't suitable for training purposes.

Reply:
Will change / update it.